TYPO3.Media/Classes/TYPO3/Media/Command/MediaCommandController.php

```diff
@@ -109,13 +124,50 @@ public function importResourcesCommand($simulate = false)
     }
 
     /**
+     * Generate thumbnails for thumbnail presets
+     *
```

Comment:
This lacks documentation for the help screen (further lines in the doc comment) which could explain when you'd want to run this command. Also a reference to clearThumbnails would make sense.

Just a remark: I noticed that you mostly use the term "generate thumbnails" in the code, but the command is called "createThumbnails". Maybe change that to "generateThumbnails" as well?